### DISCUSSION TITLE
Lazer update notification has overlapping icons

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#20227] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74088315/189514025-4d5bc3e0-65de-4115-90d9-78f894c963f0.png)
![image](https://user-images.githubusercontent.com/74088315/189514129-1dbec479-061c-41e5-860d-c28f1531ba86.png)

The download icon that is in the notification is being overlapped by the rotating loading circle icon and I'm assuming that this is a bug since it looks pretty weird.

**Comment by mares-fish** [comment]
I was about to open the same issue, glad I found it here already.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20250
